test/integration/rememberCommitMessageAfterFail/test.json

```diff
@@ -0,0 +1 @@
+{ "description": "In this test we attempt to create a commit with the commit message `test`, but a pre-commit hook fails (the hook checks for the existance of a file called `disruptive`). Afterwards we make the pre-commit hook work (by discarding the file), and again attempt to create a commit. This time the commit message should already be remembered from the failed attempt and be prefilled.", "speed": 20 }
```

Comment:
could we also add an additional step where we make another commit and verify that the old message no longer appears in the input? (i.e. type 'second commit' and if the result is 'testsecond commit' then we've failed to clear the old message)

Reply:
Done